app/views/users/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<!--
+<form_tag accept-charset="UTF-8" action="/users_path" method="post">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
+    <label for="username">Username:</label>
+    <input type="text" id="username" name="user[username]" /><br />
+    <br />
+    <label for="email">Email:</label>
+    <input type="email" id="email" name="user[email]" /><br />
+    <br />
+    <label for="email">Password:</label>
+    <input type="password" id="password" name="user[password]" /><br />
+    <br />
+    <input type="submit" value="Submit" />
+</form>
+
+<%= form_tag("/users_path", method: "post") do %>
+    <%= label_tag(:username, "Username:") %>
+    <%= text_field_tag(:username) %><br>
```

Comment:
This should be written like this 
```html
  <%= label_tag 'username' %>
  <%= text_field_tag 'username' %>
````
Note: it should be implemented for all fields .

Reply:
Done!